__tests__/app.test.js

```diff
@@ -233,5 +232,28 @@ describe("PATCH /api/articles/:article_id", () => {
         );
       });
   });
+});
+//DELETE /api/comments/:comment_id
+describe("DELETE /api/comments/:comment_id", () => {
+  test("should delete the comment and respond with status 204", () => {
+    return request(app).delete(`/api/comments/2`).expect(204);
+  });
 
+  test("should return 404 if the comment_id does not exist", () => {
+    return request(app)
+      .delete("/api/comments/345")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Comment not found");
+      });
+  });
```

Comment:
Nice error handling test, but dont forget to test for when given an endpoint that has a none existent comment_id:
E.g `/api/articles/banana`